binr/ragg2/ragg2.c

```diff
@@ -96,7 +96,11 @@ static int openfile (const char *f, int x) {
 #if __UNIX__
 	if (x) fchmod (fd, 0755);
 #endif
+#if _MSC_VER
```

Comment:
maybe, but at moment only fews truncate in all r2 code, i think we can leave here